packages/components/src/unit-control/types.ts

```diff
@@ -0,0 +1,38 @@
+export interface WPUnitControlUnit {
+	/**
+	 * The value for the unit, used in a CSS value (e.g `px`).
+	 */
+	value: string;
+	/**
+	 * The label used in a dropdown selector for the unit.
+	 */
+	label: string;
+	/**
+	 * Default value for the unit, used when switching units.
+	 */
+	default?: string | number;
+	/**
+	 * An accessible label used by screen readers.
+	 */
+	a11yLabel?: string;
+	/**
+	 * A step value used when incrementing/decrementing the value.
+	 */
+	step?: number;
+}
+
+export interface UseCustomUnitsProps {
```

Comment:
Maybe we can find a better name for this type? Or maybe we could move the types directly to the function's declaration, and add the param comments via JSDoc.

Reply:
Good idea. I've moved it over to the function declaration instead of being a separate type (since there's only one place we'll use those props). Happy to move it around again if it looks messy, though!

Reply:
That looks great, thank you!